### PR TITLE
Adding files external tests for webdav to drone

### PR DESCRIPTION
**Pull request by SergioBertolinSG**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
would be good to get coverage on files_external as well ...

File: .drone.yml [hunk 1]
```diff
@@ -254,3 +277,8 @@ matrix:
     - PHP_VERSION: 7.1
       DB_TYPE: mariadb
       TEST_SUITE: coverage
+
+    - PHP_VERSION: 7.1
+      DB_TYPE: sqlite
+      TEST_SUITE: coverage
```
Comment (DeepDiver1975) [inline .drone.yml]: If tests are executed with coverage we don't need to execute the same tests without as well

File: tests/drone/test-coverage.sh [hunk 2]
```diff
@@ -44,5 +44,8 @@ fi
 ./occ app:enable federation
 ./occ app:enable federatedfilesharing
 
-exec phpdbg -d memory_limit=4096M -rr ./lib/composer/bin/phpunit --configuration tests/phpunit-autotest.xml ${GROUP} --coverage-clover tests/autotest-clover-${DB_TYPE}.xml
-
+if [[ -z ${FILES_EXTERNAL_TYPE} ]]; then
+  exec phpdbg -d memory_limit=4096M -rr ./lib/composer/bin/phpunit --configuration tests/phpunit-autotest.xml ${GROUP} --coverage-clover tests/autotest-clover-${DB_TYPE}.xml
+else
+  exec phpdbg -d memory_limit=4096M -rr ./lib/composer/bin/phpunit --configuration tests/phpunit-autotest-external.xml ${GROUP} --coverage-clover tests/autotest-external-clover-${DB_TYPE}.xml
```
Comment (DeepDiver1975) [inline tests/drone/test-coverage.sh]: I wonder if we shall merge the config ...... let me think about that

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
there is no coverage reported for external tests - see https://codecov.io/gh/owncloud/core/pull/29661/tree/apps

File: .drone.yml (hunk 1 above)
Comment (SergioBertolinSG) [reply .drone.yml]: Removed.

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by DeepDiver1975** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! This pull request is being rebased automatically. Please DO NOT push while rebase is in progress or your changes would be lost!

**Comment by ownclouders** [comment]
Automated rebase failed! Please rebase your pull request manually via the command line.

Error:
```Command '['git', 'clone', '--depth', '1', 'rebase', '/tmp/tmp4rir7u12']' returned non-zero exit status 128.```

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by DeepDiver1975** [comment]
THX @SergioBertolinSG - please backport - THX

**Comment by SergioBertolinSG** [comment]
Backport stable10 https://github.com/owncloud/core/pull/29835

**Pull request merged**